src/agent/llm_client.py

```
@@ -0,0 +1,211 @@
+            reasoning={"effort": "medium"},
+            text={"verbosity": "medium"},
+            max_output_tokens=800,
+        )
+
+        # Prefer unified text accessor if available
+        email_content = getattr(response, "output_text", None)
+        if not email_content:
+            # Fallback to choices/message format if SDK version lacks output_text
+            try:
+                email_content = response.choices[0].message.content
+            except Exception:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)